### ISSUE TITLE
[Suggestion] Sort tracks by track number by default instead of by title

### ISSUE THREAD

**Issue by tuplasuhveli** [sailfishos-applications/flowplayer#72]
#### DESCRIPTION

Currently songs are sorted by title by default. In my opinion it would make more sense to sort them by track number by default. 

Of course one can adjust this easily in the settings, but ideally one wouldn't have to bother to do that. What do others think - which one would be expected by a new user?

**Comment by dcaliste** [comment]
I agree that sorting by track number is more natural and I would prefer to have it as the default too.
